server/main.py

```diff
@@ -48,6 +50,14 @@ async def receive(self) -> dict:
         """
         return await self._websocket.receive_json()
 
+    def __eq__(self, other: object) -> bool:
+        if not isinstance(other, Client):
+            return NotImplemented
+        return self.id == other.id
+
+    def __hash__(self) -> int:
+        return self.id.int
```

Comment:
Do we want docstrings in these? Currently, flake8 has them turned off in the `tox.ini` file.

Reply:
Good catch. I completely forgot to write docstrings for these.

Reply:
May want to remove the relevant error codes from the tox.ini file then